### ISSUE TITLE
Unable to get Vibrancy working on 

### ISSUE THREAD

**Issue by vmcilwain** [EYHN/vscode-vibrancy#94]
Hello,

I have been trying to get vibrancy installed and working for 2 days now so I thought I might ask for some help to see if it's possible at all.

I am on a mid 2015 Macbook Pro running Big Sur (11.2.3).

I did the following:

- Set my theme to Dark+ (default dark)
- Installed Vibrancy
- Reloaded Vibrancy
- Restarted VSCode
- Turned off Warning that pops up

After doing all of that, VSCode looks the same (completely solid).

I noticed in the display accessibility, reduce transparency is turned on and I don't seem to have a way to turn it off. I assume this is the reason why Vibrancy isn't working but I want to confirm that or see if there is any other workaround anyone may know about to get this working.

**Comment by Reaster0** [comment]
yup i have the same issue and i'm also running big sur, did anyone find a fix for that?

**Comment by mrAnomalyy** [comment]
Same here. (Big Sur + latest vscode version)

**Comment by boof-fo** [comment]
same on Big Sur 11.3.1

**Comment by mrAnomalyy** [comment]
Today Vibrancy just started working as well as early for me. 
Folk, try to check it out on latest updates.
<img width="400" alt="image" src="https://user-images.githubusercontent.com/39645529/119293180-ec2b2700-bc62-11eb-89d4-5ecd98f04e7a.png">


_Big Sur 11.3.1. VSCode 1.56.2_

**Comment by Reaster0** [comment]
well for me it just don't want to work even if i have exactly the same version than you, i really don't know why, do you know anything else that have change?

**Comment by mrAnomalyy** [comment]
@Reaster0 I have no idea why it works again on my computer. But I also have to mention two things:
First, I ran "Enable Vibrancy" command a few times and then restarted my VSCode.
Second, I have this line in my settings.json - `"terminal.integrated.gpuAcceleration": "off",`

Here's my settings.json:

`{
    "workbench.colorTheme": "Monokai",
    "editor.fontFamily": "'Fira Code', Menlo, Monaco, 'Courier New', monospace",
    "editor.fontLigatures": true,
    "editor.fontSize": 14,
    "editor.tabSize": 2,
    "workbench.editor.tabCloseButton": "left",
    "terminal.integrated.gpuAcceleration": "off",
    "emmet.excludeLanguages": [
        "typescript",
        "tsx",
        "jsx",
        "markdown"
    ],
}`

**Comment by Reaster0** [comment]
well i've try with vscode as root and without, with and without your configuration, still nothin, that's black magic i suppose

**Comment by Creator674** [comment]
Yo! not working fort macOS Monterey (12.1), any suggestions?

**Comment by MatiasBoldrini** [comment]
Not working on monterrey!

**Comment by JuanVillegas97** [comment]
> Not working on monterrey!

Same

**Comment by SannanOfficial** [comment]
Not working on Windows 10, firstly, I don't get the popup of "Enable Vibrancy" I get one saying "Enable iTheme Vibrancy"... whatever that means. I run that and get a popup that says

**"Command 'Enable iTheme Vibrancy' resulted an error (A system error occurred (ENOENT): no such file or directory, stat 'vscode's workbench.html path'))"**

Any thoughts?
